client/elife-theme/src/components/Button.js

```diff
@@ -0,0 +1,33 @@
+import { css } from 'styled-components'
+
+const extraSmall = css`
+  font-size: 11px;
+  line-height: calc(${props => props.theme.gridUnit} * 2);
+  padding: ${props => props.theme.gridUnit};
+`
+
+const small = css`
+  padding: ${props => props.theme.gridUnit};
+`
+
+const primary = css`
+  border: 0 none;
+`
+
+const reverse = css`
+  background: #262626;
+  color: ${props => props.theme.colorTextReverse};
+  border-color: ${props => props.theme.colorTextReverse};
+`
+
+export default css`
+  line-height: calc(${props => props.theme.gridUnit} * 4);
+  min-width: calc(${props => props.theme.gridUnit} * 16);
+  padding: calc(${props => props.theme.gridUnit} * 2);
```

Comment:
Add a comment that these three lines should be removed once the PubSweet button is updated to use the new smaller gridUnit.

Reply:
I don't think so? From my understanding the PubSweet girdUnit will change to 8px, so they will have to be updated, but not remove. I can add the update comment.

Reply:
My understanding is that when the PubSweet `Button` is updated, it will have these exact three lines. These will then be redundant as they will be duplicating rather than overriding.

Reply:
If that's what's going to happen then yes. If they will use those lines then the buttons they have now will show bigger, I was expecting them to want to keep the same sizes with different gridUnit.